cluster/src/main/java/com/hartwig/pipeline/tertiary/pave/PaveArgumentBuilder.java

```diff
@@ -0,0 +1,74 @@
+package com.hartwig.pipeline.tertiary.pave;
+
+import static com.hartwig.pipeline.metadata.InputMode.TUMOR_ONLY;
+
+import java.util.List;
+
+import com.google.api.client.util.Lists;
+import com.hartwig.pipeline.execution.vm.VmDirectories;
+import com.hartwig.pipeline.metadata.InputMode;
+import com.hartwig.pipeline.resource.RefGenomeVersion;
+import com.hartwig.pipeline.resource.ResourceFiles;
+
+public class PaveArgumentBuilder
```

Comment:
This class isn't a builder, maybe just call it PaveArguments?

Elsewhere you've inlined builders/command classes and used the class hierarchy to share common code. Would it be possible to inline this class as well for consistency?